src/res/failcode.en.json

```diff
@@ -428,6 +429,7 @@
     "lnLengthLt.pencils": "A line is shorter than the connected pencil.",
     "lnLenLt.reflect": "The lines passing a triangle are too short.",
     "lnMultipleTips.pencils": "A line connects to more than one pencil tip.",
+    "lnNoBranchOrTerm": "A line does not branch or terminate at a grid vertex",
```

Comment:
Same as Touch Slitherlink: All fail messages should end with a period.

Reply:
Fixed

Reply:
I think you missed one?

Reply:
You were right, I forgot to add `.lither` as well. Now actually fixed